boards/arduino-atmega-common/include/periph_conf.h

```diff
@@ -139,6 +139,14 @@ extern "C" {
 #endif
 /** @} */
 
+/**
+ * @name    I2C configuration
+  * @{
+ */
+#define I2C_NUMOF           1
+#define I2C_0_EN            1
+#define I2C_IRQ_PRIO        1
```

Comment:
This define can also be removed: (a) AFAIK the atmega does not support interrupt priorities, or at least RIOT does not utilize them, and (b) the define is not used anywhere in the code...